content/Api/Blip.Api.Template/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-alpine3.11 AS base
+FROM mcr.microsoft.com/dotnet/aspnet:5.0-alpine3.11 AS base
 WORKDIR /app
 EXPOSE 80
 
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1-alpine3.11 AS build
+FROM mcr.microsoft.com/dotnet/core/sdk:5.0-alpine3.11 AS build
```

Comment:
Same about the docker image here.

Reply:
In this case, mcr.microsoft.com/dotnet/core/sdk:5.0 has moved to dotnet/sdk. I will upgrade this line of code.